lib/cmd-utils/get-contextual-tuples.go

```diff
@@ -0,0 +1,33 @@
+package cmdutils
+
+import (
+	"strings"
+
+	"github.com/openfga/cli/lib/clierrors"
+	"github.com/openfga/go-sdk/client"
+	"github.com/spf13/cobra"
+)
+
+func ParseContextualTuples(cmd *cobra.Command) ([]client.ClientTupleKey, error) {
+	contextualTuples := []client.ClientTupleKey{}
+	contextualTuplesArray, _ := cmd.Flags().GetStringArray("contextual-tuple")
+
+	if len(contextualTuplesArray) > 0 {
+		for index := 0; index < len(contextualTuplesArray); index++ {
```

Comment:
do we want to increase index by 1 or by 3?

Reply:
1, each tuple is a single entity we parse later

```sh
fga query list-objects --store-id=01H0H015178Y2V4CX10C2KGHF4 user:anne can_view document --contextual-tuple "user:anne can_view folder:product" --contextual-tuple "folder:product parent document:roadmap"
```